### PR TITLE
Replace old certificationText with certification attri. in print

### PR DESCRIPTION
**Pull request by marionb**
GSOREB-249
Last point in https://jira.camptocamp.com/projects/GSOREB/issues/GSOREB-249
Consider the following:
- I did not add the `CertificationAtWeb` as it is not used for the moment --> see issue.
- https://github.com/camptocamp/pyramid_oereb/blob/master/doc/source/changes.rst contains some reference to the old attribute `certificationText` we should probability (re-)move this?

### CODE REVIEW

**Comment by ger-benjamin** [comment]
> I did not add the CertificationAtWeb as it is not used for the moment --> see issue.

:ok_hand: 

> https://github.com/camptocamp/pyramid_oereb/blob/master/doc/source/changes.rst contains some reference to the old attribute certificationText we should probability (re-)move this?

This will be done with https://jira.camptocamp.com/browse/GSOREB-248 I think

File: print/print-apps/oereb/mainpage.jrxml [hunk 1]
```diff
@@ -36,7 +36,7 @@
 	<parameter name="mapSubReport" class="java.lang.String"/>
 	<parameter name="scalebarSubReport" class="java.lang.String"/>
 	<parameter name="northArrowSubReport" class="java.lang.String"/>
-    <parameter name="certificationText" class="java.lang.String"/>
+    <parameter name="Certification" class="java.lang.String"/>
```
[the remark this answers is not in the document]
Comment (marionb) [reply print/print-apps/oereb/mainpage.jrxml]: Hmm this is strange: opening the document with vim locally I don't see anything wrong with the indentation (just here in the browser), so I can not correct it :-/

[the remark this answers is not in the document]
Comment (ger-benjamin) [reply print/print-apps/oereb/mainpage.jrxml]: Perhaps it's a problem with "space indentation" and "tab" indentation. In this case, you'll get a travis error

**Comment by jwkaltz** [comment]
The title of the PR should be the opposite, right?
We are replacing the old (CertificationText) with the new (Certification).

**Pull request merged**